tensorflow/lite/python/util.py

```diff
@@ -110,6 +110,9 @@ def get_tensors_from_tensor_names(graph, tensor_names):
   tensors = []
   invalid_tensors = []
   for name in tensor_names:
+    if not isinstance(name, str):
+      raise ValueError("Invalid tensor name '{}' was found".format(name))
```

Comment:
third_party/tensorflow/lite/python/util.py", line 120, in get_tensors_from_tensor_names
    raise ValueError("Invalid tensor name '{}' was found".format(name))
ValueError: Invalid tensor name 'input' was found

Reply:
I've just posted a new version of PR where I print a type as well. Could you please re-run and post its output?